content/hi/auto_scaling.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ऑटो स्केलिंग
+status: Completed
+category: गुण
+---
+
+ऑटोस्केलिंग एक सिस्टम की स्वचालित रूप से [स्केल](/scalability/) होने की क्षमता है, आमतौर पर, कंप्यूटिंग संसाधनों के संदर्भ में। एक ऑटोस्केलिंग सिस्टम के साथ, जरूरत पड़ने पर संसाधन स्वचालित रूप से जुड़ जाते हैं और उपयोगकर्ता की उतार-चढ़ाव वाली मांगों को पूरा करने के लिए स्केल कर सकते हैं। ऑटोस्केलिंग प्रक्रिया भिन्न होती है और विभिन्न मेट्रिक्स, जैसे मेमोरी या प्रक्रिया समय के आधार पर स्केल करने योग्य होती है। प्रबंधित क्लाउड सेवाएं आमतौर पर ऑटोस्केलिंग कार्यक्षमता से जुड़ी होती हैं क्योंकि अधिकांश ऑन-प्रिमाइसेस परिनियोजन की तुलना में अधिक विकल्प और कार्यान्वयन उपलब्ध होते हैं।
```

Comment:
We could localize memory to `स्मरणशक्ति` maybe, open to thoughts from team.

Reply:
It will make it much difficult to understand we are talking about memory. (memory as in computer memory, not human memory). In my opinion, it should be kept मेमोरी